R/keep.R

```diff
@@ -1,7 +1,7 @@
 #' Simple function to get only the pregnancy event dates.
 #'
-#' @param lpr2 Output from `join_lpr2()`.
-#' @param lpr3 Output from `join_lpr3()`.
+#' @param lpr2 Output from `prepare_lpr2()`.
+#' @param lpr3 Output from `prepare_lpr3()`.
```

Comment:
```suggestion
#' @param lpr2 Output from [prepare_lpr2()].
#' @param lpr3 Output from [prepare_lpr3()].
```

To auto-link to the documentation of those functions.